### PR TITLE
Manifestv2

### PR DESCRIPTION
**Pull request by Salamandar**

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!bookwormtestme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "digiscreen"
+name = "Digiscreen"
+description.en = "An interactive wallpaper for the classroom, face-to-face or distant"
```
Comment (ericgaspar) [inline manifest.toml:7]:
```suggestion
description.en = "Interactive wallpaper for the classroom, face-to-face or distant"
```

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+description.fr = "Un fond d'écran interactif pour la salle de classe en présence ou à distance."
```
Comment (ericgaspar) [inline manifest.toml:8]:
```suggestion
description.fr = "Fond d'écran interactif pour la salle de classe en présence ou à distance"
```

File: manifest.toml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+license = "GPL-3.0-only"
+website = "https://ladigitale.dev/"
+demo = "https://ladigitale.dev/digiscreen/"
+admindoc = "https://codeberg.org/ladigitale/digiscreen/src/branch/main/README.md"
+userdoc = "https://ladigitale.dev/blog/digiscreen-un-fond-d-ecran-interactif-pour-la-classe"
+code = "https://codeberg.org/ladigitale/digiscreen"
+fund = "https://opencollective.com/ladigitale"
+
+[integration]
+yunohost = ">= 11.0.10"
```
Comment (ericgaspar) [inline manifest.toml:24]:
```suggestion
yunohost = ">= 11.2"
```

File: manifest.toml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    help.en = "Optional: a Pixabay API key will let you chose a background by entering keywords."
+    help.fr = "Optionnel : une clé d’API Pixabay vous permettra de choisir un fond d’écran en entrant un mot clé."
+    type = "string"
+    example = "123456abc"
+    optional = true
+
+    [install.apikey_google_youtube]
+    ask.en = "Your Google API key"
+    ask.fr = "Votre clé d’API Google"
+    help.en = "Optional: a Google API key will let you insert videos from Youtube on your Digiscreen"
```
Comment (ericgaspar) [inline manifest.toml:53]:
```suggestion
    help.en = "Optional: a Google API key will let you insert videos from YouTube on your Digiscreen"
```

File: manifest.toml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    help.fr = "Optionnel : une clé d’API Google vous permettra d’afficher des vidéos de Youtube sur votre Digiscreen."
```
Comment (ericgaspar) [inline manifest.toml:54]:
```suggestion
    help.fr = "Optionnel : une clé d’API Google vous permettra d’afficher des vidéos de YouTube sur votre Digiscreen."
```

File: manifest.toml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    type = "string"
+    example = "123456abc"
+    optional = true
+
+    [install.init_main_permission]
+    help.en = "If public, any visitor can use this app and thus will benefit from your API keys for Pixabay and Youtube."
```
Comment (ericgaspar) [inline manifest.toml:60]:
```suggestion
    help.en = "If public, any visitor can use this app and thus will benefit from your API keys for Pixabay and YouTube."
```

File: manifest.toml [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    help.fr = "Si l’application est publique, n’importe quel visiteur pourra utiliser cette application et bénéficiera donc de vos clés d’API pour Pixabay et Youtube."
```
Comment (ericgaspar) [inline manifest.toml:61]:
```suggestion
    help.fr = "Si l’application est publique, n’importe quel visiteur pourra utiliser cette application et bénéficiera donc de vos clés d’API pour Pixabay et YouTube."
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: manifest.toml (hunk 7 above)
Comment (ericgaspar) [reply manifest.toml:61]: Also: `php-fpm.conf` -> `extra_php-fpm.conf` or removed like befor

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**